README.md

```diff
@@ -289,6 +289,19 @@ params.SourceRepo = "source-repo"
 rtManager.DownloadFiles(params)
 ```
 
+#### Promoting a docker image in Artifactory
+
+```go
+params := services.NewDockerPromoteParams(sourceRepo, targetRepo, dockerRepository)
```

Comment:
I think that the name of the third argument here (dockerRepository) is pretty confusing. I suggest renaming it to sourceDockerImage and also add an example or two.